extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -185,14 +183,14 @@ private function getUploadErrorMessage( $status ) {
 	 *
 	 * @author Federico "Lox" Lucignano
 	 */
-	private function getUploadWarningMessages( $warnings ){
-		$ret = array();
+	private function getUploadWarningMessages( $warnings ) {
+		$ret = [];
 
-		foreach( $warnings as $warning => $args ) {
+		foreach ( $warnings as $warning => $args ) {
 			if ( $args === true ) {
-				$args = array();
+				$args = [];
 			} elseif ( !is_array( $args ) ) {
-				$args = array( $args );
+				$args = [ $args ];
 			}
 
 			$ret[] = wfMsgExt( $warning, 'parseinline', $args );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)